proto/osmosis/pool-incentives/v1beta1/gov.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+package osmosis.poolincentives.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "osmosis/pool-incentives/v1beta1/incentives.proto";
+
+option go_package = "github.com/c-osmosis/osmosis/x/pool-incentives/types";
+
+// AddPoolIncentivesProposal is a gov Content type for adding the pool incentives.
+// If a AddPoolIncentivesProposal passes, the proposal’s record is be added to the module.
+// Each record has a specified pot id and weight, and the incetives are distributed to each pot according to weight/total_weight.
```

Comment:
would be good to add where the income source is from, not only distribution rule.

Reply:
Added text about that